examples/tts/conf/fastpitch/fastpitch.yaml

```diff
@@ -193,6 +193,8 @@ model:
   alignment_module:
     _target_: nemo.collections.tts.modules.aligner.AlignmentEncoder
     n_text_channels: ${model.symbols_embedding_dim}
+    dist_type: cosine
```

Comment:
nit: it seems we now only support L2 and Cosine that are hidden in the aligner.py. It would be great to mention supported ones as a comment.

Reply:
Added documentation to the aligner constructor.